resources/scss/views/ItemList/Components/Filter/item-filter-list.scss

```diff
@@ -1,8 +1,97 @@
-.findologic-filter-wrapper #filterCollapse {
-    position: relative;
-    z-index: 0;
-}
+.list-controls {
+    #filterCollapse {
+        position: relative;
+        z-index: 0;
+    }
+    .main-filters {
+        position: relative;
+        z-index: 0;
+        width: 100%;
+        margin-left: -15px;
+        padding-right: 15px;
+        top: 100%;
 
-.list-controls #filterCollapse .page-content .card .form-check .filter-badge {
-    top: 1.5em;
+        .page-content {
+            background: white;
+            padding: 2em !important;
+            border: 1px solid #F4F4F4;
```

Comment:
Aren't there any predefined variables for all these colors?

Reply:
I have made that there is one main Findologic.scss helper file that would import all other needed scss files. And Findologic is imported inside item-filter-list.scss.

I think a better solution would be to have one main Findologic.scss file that would import all needed scss files from other modules (Ceres) and import local scss (item-filter-list.scss). 
But for this we would need to rewrite gulp task, because right now it compiles all scss files found in resources/scss/ folder